Makefile

```diff
@@ -29,15 +29,16 @@ install: earlyoom.service install-bin install-default
 install-initscript: earlyoom.initscript install-bin install-default
 	install -d $(DESTDIR)$(SYSCONFDIR)/init.d/
 	cp $< $(DESTDIR)$(SYSCONFDIR)/init.d/earlyoom
-	chmod a+x $(DESTDIR)$(SYSCONFDIR)/init.d/earlyoom
+	chmod 0755 $(DESTDIR)$(SYSCONFDIR)/init.d/earlyoom
```

Comment:
I wonder if we should use "install" instead of cp + chmod?

Reply:
install just can't rename files

Reply:
I think it can:
```
$ touch foo
$ install foo foo2
$ ls -l
total 0
-rw-rw-r--. 1 jakob jakob 0  2. Nov 08:40 foo
-rwxr-xr-x. 1 jakob jakob 0  2. Nov 08:41 foo2
```